### ISSUE TITLE
INSTALL.md Formatting 

### ISSUE THREAD

**Issue by braughtg** [DickinsonCollege/FarmData2#44]
The formatting of the INSTALL.md file needs to be improved.  In particular there some lists that do not continue appropriately near the end of the document.  This would require studying the GitHub markdown syntax a bit and then updating the INSTALL.md file to improve the format.

**Comment by braughtg** [comment]
Some of the major issues were addressed. There are still issues with list numbering and nested lists. Some lists also appear as highlighted (quoted?) blocks but should not.

**Comment by amelia291** [comment]
The headings have different sizes, and since they are not indented or anything, I assume that none of them are subheadings. If they are of the same level, they should be of the same size.

**Comment by IrisSC** [comment]
Under the last section "Stopping and Starting FarmData2" there is a typo. In the first sentence FarmData2 is reference by it is spelled "FarmDat2" and needs to be changed to "FarmData2"

**Comment by Batese2001** [comment]
Most of the second half of the document did not number correctly. Instead, it would repeat "1." rather than continuing.

**Comment by boosungkim** [comment]
As mentioned, I noticed a problem with the numbering in this markdown file. I would also put the last sentence in the file, "Thus, the newly created containers...," in a new line, rather than continuing on the same bullet point.

**Comment by SavidBasnyat** [comment]
As mentioned above, I noticed that the numbering was not correct for "Bringing Up FarmOS" and "Importing Sample Data" sections.

**Comment by Diego2214** [comment]
I noticed that there is a typo in the "Bringing Up FarmOS" in the first command we are asked to use. There should not be a "sudo" in the command. Just as an add-on, there should also be a sentence stating that the tag "-d" in the same command returns the command prompt.

**Comment by ShannonLoughin** [comment]
> As mentioned above, I noticed that the numbering was not correct for "Bringing Up FarmOS" and "Importing Sample Data" sections.

I also noticed this issue.

**Comment by zolboo1024** [comment]
> As mentioned above, I noticed that the numbering was not correct for "Bringing Up FarmOS" and "Importing Sample Data" sections.

I also noticed this issue. The install steps in the section "Bringing Up FarmOS" are all numbered "1" instead of 1,2,3,4,5

**Comment by josieecook** [comment]
> Most of the second half of the document did not number correctly. Instead, it would repeat "1." rather than continuing.

I noticed this too. It was also not indented correctly, and was instead all one block of text.

**Comment by mollerup23** [comment]
Like other have mentioned, I noticed a numbering issue in the last markdown box under the 'Importing Sample Data' field. Additionally, when selecting the 'FarmData2/docker/farmdata2files/farmdata2dbpartII.sql' to upload, the installation guide fails to mention you must click 'Open' in order to choose the file before clicking the 'Go' button to upload it.

**Comment by hantrinh13** [comment]
In the section "Adding FarmData2 Customizations," in the sublists of "Copy the FarmData2 logo..." and "Copy the FarmData2 custom modules...," the numbering is not correct (they are all 1's instead of 1 2 3 4 ...). Also, these points are written on the same line/paragraph, which is very hard to follow. It is better if each step is separated from each other.

**Comment by miyu386** [comment]
As mentioned above, list items in the later part of the page have repeated 1's instead of having a properly numbered list.

**Comment by won369369** [comment]
typo at the third sentence of a Development platform section. (“The it’s”)

**Comment by infantlikesprogramming** [comment]
The number formating for some of the steps are wrong. For example, in Bringing Up FarmOS - Use phpMyAdmin to add a new database user:, the steps are not correctly formatted.

**Comment by egorovak** [comment]
> The number formating for some of the steps are wrong. For example, in Bringing Up FarmOS - Use phpMyAdmin to add a new database user:, the steps are not correctly formatted.

I have noticed this issue too with the second part of the document

**Comment by vuphuongha** [comment]
some lines are hard to read because there are no "," (commas), so it makes the whole sentences sound weird and long. I also notice that there are all 1's in the Bringing Up FarmOS section and Importing Sample Data section like others who have also addressed this issue.

**Comment by brunokaboyi** [comment]
The number formatting in the paragraph that starts with "Copy the FarmData2 logo into the volume mounted..." is off. The list is numbered with 1s and i personally feel like it would look better and easier to read if the steps had been put into a bulleted list

**Comment by braughtg** [comment]
Currently working on an new, cleaner, simpler, easier, install process.  I will update INSTALL.md to match.
